custom-remote-user-store-management/manager/src/main/java/org/wso2/carbon/remote/user/store/manager/internal/WSUserStoreComponentHolder.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.wso2.carbon.remote.user.store.manager.internal;
+
+import org.wso2.carbon.user.core.service.RealmService;
+
+public class WSUserStoreComponentHolder {
+
+    private static WSUserStoreComponentHolder instance = new WSUserStoreComponentHolder();
+    private RealmService realmService;
+
+    public static WSUserStoreComponentHolder getInstance() {
+
+        return instance;
+    }
+
+    public RealmService getRealmService() {
+
+        return realmService;
+    }
+
+    public void setRealmService(RealmService realmService) {
+
+        this.realmService = realmService;
+    }
+
```

Comment:
Please remove this line.

Reply:
Fixed in 51041691a23b06016ed6fa558504cc92b89c40b3 and 8a4b4fe793b231e0bd48d328c076d4cfd9494c5e